doc/components.md

```diff
@@ -90,15 +90,65 @@ class GameOverPanel extends PositionComponent with HasGameRef<MyGame> {
 
 ## PositionComponent
 
-This class represent a single object on the screen, being a floating rectangle or a rotating sprite.
+This class represent a positioned object on the screen, being a floating rectangle or a rotating
+sprite. It can also represent a group of positioned components if children are added to it.
 
-A `PositionComponent` has a `position`, `size` and `angle`, as well as some useful methods like
-`distance` and `angleBetween`.
+The base of the `PositionComponent` is that it has a `position`, `size`, `scale`, `angle` and
+`anchor` which transforms how the component is rendered.
+
+### Position
+
+The `position` is just a `Vector2` which represents the position of the component in relation to its
+parent, if the parent is a `FlameGame` it is in relation to the viewport.
+
+### Size
+
+The `size` is the size of the component when the zoom level of the camera is 1.0 (no zoom, default).
+The `size` is not in relation to the parent of the component.
+
+### Scale
+
+The `scale` is how much the component and its children should be scaled and since it is represented
+by a `Vector2` you can scale in uniform way by changing `x` and `y` with the same amount, if you
+want to scale in a non-uniform way, change `x` or `y` more than the other.
```

Comment:
```suggestion
The `scale` is how much the component and its children should be scaled. Since it is represented
by a `Vector2`, you can scale in a uniform way by changing `x` and `y` with the same amount, or in a non-uniform way, by change `x` or `y` by different amounts.
```